src/index.js

```diff
@@ -0,0 +1,55 @@
+import './style.css';
+import './imgs/bg1.png';
+
+const addTask = document.querySelector('.addTask');
+const newDest = document.querySelector('.newToDo');
+const addBtn = document.querySelector('.add_button');
+const remBtn = document.querySelector('.remove_button');
+let task = [];
+
+function addlist() {
+  task.push(newDest.value);
+}
+
+function showList() {
+  addTask.innerHTML = '';
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < task.length; i++) {
+    const container = `
+    <div class ="squareT">
+      <div class ="square">
+        <ul class="toDo">
+          <li>
+            <input 
+            type="checkbox"
+            class="checkbox" 
+            name="checkbox" 
+            id="checkbox" ></li>
+          <li class="task-description">${task[i]}</li>
+        </ul>
+      </div>
```

Comment:
- [ ] Great hob working on the design and structure but I think the requirements is to display a list contain some tasks so you can create as required an array of objects and just make a function to iterate through the tasks 💯✨
![image](https://user-images.githubusercontent.com/58553711/206056311-dd7eb3e8-d49a-426e-a793-0ddb85615392.png)